extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/util/MultiStageQueryContext.java

```diff
@@ -380,6 +380,10 @@ static IndexSpec decodeIndexSpec(@Nullable final Object indexSpecObject, final O
    */
   public static TaskLockType validateAndGetTaskLockType(QueryContext queryContext, boolean isReplaceQuery)
   {
+    final boolean useConcurrentLocks = queryContext.getBoolean(Tasks.USE_CONCURRENT_LOCKS, false);
```

Comment:
Capture the default value of `false` in a single place, preferably `Tasks` class itself.

Reply:
Done